packages/dashboards/app/styles/navi-dashboards/components/navi-collection.less

```diff
@@ -35,30 +35,27 @@
     }
   }
 
-  .actions {
-    display: flex;
-
-    button,
-    .btn,
-    a {
-      padding: 0;
-
-      i {
-        color: @navi-gray;
-        font-size: @font-size-base;
-
-        &:hover {
-          color: @navi-dark-gray;
-        }
-      }
+  .dashboard-action-list {
+    .modal.is-active {
+      visibility: visible;
     }
+  }
 
-    .action {
-      margin-right: 10px;
+  &__row{
+    height: 42px;
+  
+    .d-icon {
+      visibility: hidden;
+    }
 
-      .modal.is-active {
-        visibility: visible;
+    &:hover {
+      .d-icon {
+        visibility: visible !important;
       }
     }
   }
+
+  &__actions {
+    width: 294px;
```

Comment:
Magic number? Do we need a set width?

Reply:
This prevents the content shift when hovering, because previously actions was given only a small margin of space and then reallocated more when hovered